Bio/motifs/mast.py

```diff
@@ -91,19 +91,23 @@ def __read_database_and_motifs(record, handle):
     elif words[2] == '(peptide)':
         record.alphabet = IUPAC.protein
     for line in handle:
-        if 'MOTIF WIDTH' in line:
+        if 'WIDTH BEST POSSIBLE MATCH' in line:
```

Comment:
Should this be looking for either the old form, or the new form?

Or is ``'WIDTH BEST POSSIBLE MATCH'`` present in both, making it a better place to break?

Reply:
That's correct, `'WIDTH BEST POSSIBLE MATCH'` should be present in the output file regardless of version. See outputs for version [4.11.4](http://web.mit.edu/meme_v4.11.4/share/doc/examples/mast_example_output_files/mast.txt) and  [3.0](https://github.com/biopython/biopython/blob/ce904f/Tests/motifs/mast.dna.oops.txt#L32).